ios_tests/lib/ios/specs/device/touch_actions.rb

```diff
@@ -1,5 +1,19 @@
 describe 'device/touch_actions' do
-  t {} # place holder test
+  t 'swipe_default_duration' do
+    wait_true do
+      text('pickers').click
+      screen == 'Pickers'
+    end
+
+    ele_index('UIAStaticText', 2).text.must_equal 'John Appleseed - 0'
+    picker = ele_index('UIAPicker', 1)
+    loc = picker.location.to_h
+    size = picker.size.to_h
+    start_x = loc[:x] + size[:width] / 2
+    start_y = loc[:y] + size[:height] / 2
+    swipe start_x: start_x, start_y: start_y, end_x: start_x, end_y: - 50
```

Comment:
I am not sure why in iOS end coordinates are relative to start coordinates, is that a known issue in appium server?